apps/authenticator/lib/sign_in/commands/resource_owner.ex

```diff
@@ -96,6 +91,11 @@ defmodule Authenticator.SignIn.Commands.ResourceOwner do
         FakeVerifyHash.execute(:argon2)
         {:error, :unauthenticated}
 
+      {:public?, false} ->
+        Logger.info("Client application #{client_id} is not public")
+        FakeVerifyHash.execute(:argon2)
+        {:error, :unauthenticated}
```

Comment:
Here i'll request the `client_assertion` in another pull request.

Reply:
Ok

Reply:
Failed  code consistence: 
https://github.com/lcpojr/watcher_ex/pull/53/checks?check_run_id=2774855844